doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
That's due to clobbering by `Dates` again, same issue as above. It's not [lost in helpdb](https://github.com/JuliaLang/julia/blob/c742260b589799c5ae42e61b8606d6f67603479c/base/docs/helpdb.jl#L580-L588) so can be recovered.